widgets/hits-per-page-selector/hits-per-page-selector.js

```diff
@@ -41,14 +41,14 @@ function hitsPerPageSelector({
   return {
     init: function(state) {
       let isCurrentInOptions = reduce(options, function(res, option) {
```

Comment:
This is an incorrect use of `reduce`. It doesn't stop as soon as it find the value, and it is not explicit. A good alternative could `_.includes`.